src/title-nav/block.spec.js

```diff
@@ -0,0 +1,144 @@
+// title-nav test
+jest.setTimeout(30000)
+import {
+    createNewPost,
+    enablePageDialogAccept,
+    getEditedPostContent,
+    insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+import {
+    clickElementByText,
+    blockStartup,
+    updateRangeInput,
+    clickCheckbox,
+    openSidebarPanelWithTitle
+} from '../test-helpers'
+
+const block = {blockTitle: 'Titled Navigation', blockName: 'purdue-blocks/title-nav'}
+
+describe( 'Titled Navigation Block', () => {
+    beforeAll( async () => {
+        await enablePageDialogAccept();
+    } );
+    beforeEach( async () => {
+        await createNewPost();
+    } );
+
+    it( 'Block should be available.', async () => {
+        await insertBlock( 'Titled Navigation' )
+
+
+        // tests that the block is properly inserted and matches the existing snapshot
+        expect(await page.$('[data-type="purdue-blocks/title-nav"]')).not.toBeNull()
+        expect( await getEditedPostContent() ).toMatchSnapshot()
```

Comment:
more of a question than anything, but i see a lot of await statements. are these all needed?

Reply:
found what i was looking for with await 
```
test('the data is peanut butter', async () => {
  await expect(fetchData()).resolves.toBe('peanut butter');
});

test('the fetch fails with an error', async () => {
  await expect(fetchData()).rejects.toThrow('error');
});
```

it looks like in the docs they slap await outside of the expect block not sure if it will accomplish anything but it looks more pretty